sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/ConfigurationClientBuilder.java

```diff
@@ -231,6 +252,24 @@ public ConfigurationClientBuilder connectionString(String connectionString) {
 
         this.endpoint = credential.getBaseUri();
 
+        // Make sure only use connection string
+        this.tokenCredential = null;
+        return this;
+    }
+
+    /**
+     * Sets the AAD credential to use when authenticating HTTP requests.
+     *
+     * @param tokenCredential AAD token
```

Comment:
Similar comment as above:

```suggestion
     * @param tokenCredential TokenCredential used to authenticate HTTP requests.
```